Add an option to force the build or not of corba dependend stuff

--- CMakeLists.txt.orig	2013-08-22 14:42:19.936889278 +0200
+++ CMakeLists.txt	2013-08-22 14:44:37.632894513 +0200
@@ -199,6 +199,11 @@
   SET(BUILD_RTALLOC OFF CACHE BOOL "docstring" FORCE)
 ENDIF()
 
+OPTION( BUILD_CORBA "Build with corba support (require corba support in RTT);" TRUE)
+IF(NOT BUILD_CORBA)
+  SET(OROCOS-RTT_CORBA_FOUND FALSE)
+ENDIF()
+
 
 
 ###########################################################
